### ISSUE TITLE
Possible Bug

### ISSUE THREAD

**Issue by XinYu-Andy** [threestudio-project/threestudio#63]
Hi, thank you for your well-structured code. I've noticed that each time it runs, it defaults to backing up the code from the current run to the experiment storage files, seemingly excluding the "outputs" folder. However, if I change exp_root_dir in the config file, say, to "exp_results", it causes a loop that stores the results in this folder, leading to an exponential increase in memory usage. Could you advise me on how I should modify the code?

**Comment by XinYu-Andy** [comment]
Seems I can manually change .gitignore to avoid it.

**Comment by bennyguo** [comment]
@XinYu-Andy Yes the CodeSnapshotCallback saves all git-tracked files (except for `load/`). This could be a problem, I'll reopen this issue and mark it as "enhancement". Thanks for reporting!

**Comment by claforte** [comment]
Not sure if this bug has been addressed... can we close it?

**Comment by thuliu-yt16** [comment]
Not yet and no elegant method has been thought of yet

> Not sure if this bug has been addressed... can we close it?
